packages/react-native-editor/__device-tests__/gutenberg-editor-block-insertion-2.test.js

```diff
@@ -57,47 +55,17 @@ describe( 'Gutenberg Editor tests for Block insertion 2', () => {
 		expect( html.toLowerCase() ).toBe( expectedHtml );
 	} );
 
-	it( 'inserts block before selected block', async () => {
```

Comment:
I checked the rest of `Gutenberg Editor tests for Block insertion 2` E2E tests and seems that we could also migrate them to integration tests, plus none of them are using a native-specific functionality 🎊.

Reply:
I agree 😎 , since they're only 4 left I can move them in a separate PR so have one less E2E test, what do you think?